### DISCUSSION TITLE
Could `InferCreationAttributes` exclude properties with `Default` decorator ?

### DISCUSSION THREAD

**Discussion by pikadun** [sequelize/sequelize#15747] [Help & Questions] [answered]
This will make work easier.

**Comment by ephys** [comment, marked as answer]
You need to wrap them with `CreationOptional`. It's currently impossible to modify the type of a property with decorators

We made that more explicit in the upcoming v7 docs: https://6404627a0dfed06840d8efb6--sequelize-site.netlify.app/docs/v7/models/defining-models/#default-values
